lib/recurly/http_adapter.php

```diff
@@ -8,6 +8,9 @@
 
 namespace Recurly;
 
+/**
+ * @codeCoverageIgnore
```

Comment:
Is there a reason why we want to ignore?

Reply:
This `HttpAdapter` class was created specifically so that it could be excluded from the testing framework. Anything that would test this class would be implicitly testing PHP's core http framework and we couldn't design any tests without being dependent on an external http server.